packages/client/components/assignment-selections/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+  Button
+} from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import styled from 'styled-components'
+
+const DetailContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const ButtonContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
I think `width: 100%` is redundant here, unless there's a good reason for it.

Reply:
Also, again, I think `Grid` could be used here as well unless there's a reason not to.